src/sentry/static/sentry/app/components/group/pluginActions.jsx

```diff
@@ -0,0 +1,232 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import Modal from 'react-bootstrap/lib/Modal';
+import ApiMixin from 'app/mixins/apiMixin';
+import DropdownLink from 'app/components/dropdownLink';
+import GroupState from 'app/mixins/groupState';
+import MenuItem from 'app/components/menuItem';
+import plugins from 'app/plugins';
+import {t} from 'app/locale';
+import {toTitleCase} from 'app/utils';
+import IssueSyncListElement from 'app/components/issueSyncListElement';
+
+
+const PluginActions = createReactClass({
+  displayName: 'PluginActions',
+
+  propTypes: {
+    group: PropTypes.object.isRequired,
+    plugin: PropTypes.object.isRequired,
+  },
+
+  mixins: [ApiMixin, GroupState],
+
+  getInitialState() {
+    return {
+      showModal: false,
+      actionType: null,
+      issue: null,
+      pluginLoading: false,
+    };
+  },
+
+  componentWillMount() {
+    this.loadPlugin(this.props.plugin);
+  },
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.plugin.id !== nextProps.plugin.id) {
+      this.loadPlugin(nextProps.plugin);
+    }
+  },
+
+  ACTION_LABELS: {
+    create: t('Create New Issue'),
+    link: t('Link with Existing Issue'),
+    unlink: t('Unlink Issue'),
+  },
+
+  deleteIssue() {
+    const plugin = this.props.plugin;
+    // override plugin.issue so that 'create/link' Modal
+    // doesn't think the plugin still has an issue linked
+    Object.assign(plugin, {
+      issue: null
+    });
+    const endpoint = `/issues/${this.props.group.id}/plugins/${
+      this.props.plugin.slug
+    }/unlink/`;
+    this.api.request(endpoint, {
+      success: data => {
+        this.loadPlugin(plugin);
+      },
+      error: error => {},
```

Comment:
I haven't been following the component flows here but do we need an error message here?